src/components/v2v3/shared/FundingCycleConfigurationDrawers/DistributionSplitModal/V2ProjectPayoutFormItem.tsx

```diff
@@ -0,0 +1,62 @@
+import { t, Trans } from '@lingui/macro'
+import * as constants from '@ethersproject/constants'
+import { Form, InputNumber, Radio } from 'antd'
+import { stringIsDigit } from 'utils/math'
+import { featureFlagEnabled } from 'utils/featureFlags'
+import { FEATURE_FLAGS } from 'constants/featureFlags'
+
+export const V2_ALLOCATOR_ADDRESS = 'TODO_V2_ALLOCATOR_ADDRESS'
+const V3_ALLOCATOR_ADDRESS = constants.AddressZero
```

Comment:
**suggestion**: perhaps `constants/contracts/mainnet` is a better place for these, WDYT?